http/codegen/server.go

```diff
@@ -823,17 +823,14 @@ func {{ .ErrorEncoder }}(encoder func(context.Context, http.ResponseWriter) goah
 	return func(ctx context.Context, w http.ResponseWriter, v error) {
 		switch res := v.(type) {
 
-		{{- range $errs := .Errors }}
-		case {{ .Ref }}:
+	{{- range $ref := .Errors.Refs }}
+		case {{ $ref }}:
+			{{- range $.Errors.Get $ref }}
 
-	       {{- with .Response}}
-			{{- if .TagName }}
-				{{- if .TagRequired }}
+	      {{- with .Response}}
```

Comment:
indenting seems off...

Reply:
weird why I am having indenting issues.. never had this problem before.. probably an editor update messed up with my settings.